### PR TITLE
SLING-10528 - Reject large packages

### PR DESCRIPTION
**Pull request by tmaret**

### CODE REVIEW

File: src/main/java/org/apache/sling/distribution/journal/bookkeeper/BookKeeper.java [hunk 1]
```diff
@@ -85,6 +85,7 @@
     private static final String SUBSERVICE_IMPORTER = "importer";
     private static final String SUBSERVICE_BOOKKEEPER = "bookkeeper";
     private static final int RETRY_SEND_DELAY = 1000;
+    static final long MAX_PACKAGE_SIZE = 5242880; // 5MB
```
Comment (joerghoh) [inline src/main/java/org/apache/sling/distribution/journal/bookkeeper/BookKeeper.java:88]: Shouldn't we make that configurable? 5 MB seems to be a reasonable default though.

**Review by joerghoh — CHANGES_REQUESTED** [review]

File: src/main/java/org/apache/sling/distribution/journal/bookkeeper/BookKeeper.java (hunk 1 above)
Comment (tmaret) [reply src/main/java/org/apache/sling/distribution/journal/bookkeeper/BookKeeper.java:88]: We discussed that in the ticket and concluded that it'd be a compatible change to add a configuration in the future, should a configuration be needed.

**Comment by joerghoh** [comment]
then fine for me.

**Comment by tmaret** [comment]
Thanks @joerghoh ! Does that count as an approval ?

**Review by joerghoh — APPROVED** [review]

**Pull request merged**